code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -198,3 +198,27 @@
 		locked = FALSE
 		icon_state = icon_broken
 		origin_tech = null //wipe out any origin tech if it's unlocked in any way so you can't double-dip tech levels at R&D.
+
+/obj/item/storage/lockbox/experimental_weapon
+	name = "A-113 classified lockbox"
+	desc = "Contains a classifed item for experimental purposes. Looks like some acid was spilt on it."
+	req_access = list(ACCESS_SEC_DOORS) //officers, heads
+
+/obj/item/storage/lockbox/experimental_weapon/populate_contents()
+	if(prob(10))
+		new /obj/item/clothing/mask/facehugger(src) //Suprise! Storing facehuggers improperly is what lead to this mess.
+		return
+	var/spawn_type = pick(/obj/item/gun/energy/kinetic_accelerator/experimental, /obj/item/surveillance_upgrade, /obj/item/mod/module/stealth/ninja)
+	if(prob(25))
+		if(rand(1, 6) == 1) //organ time. I want this to be more balanced in distribution, so organs are under a prob 25
+			new /obj/item/organ/internal/alien/plasmavessel/drone(src)  //Disected drone before the place got wiped. No hivenode.
+			new /obj/item/organ/internal/alien/acidgland(src)
+			new /obj/item/organ/internal/alien/resinspinner(src)
+			return
+		spawn_type = pick(
+			/obj/item/organ/internal/cyberimp/arm/katana,
+			/obj/item/organ/internal/cyberimp/arm/toolset_abductor,
+			/obj/item/organ/internal/cyberimp/arm/esword,
+			/obj/item/organ/internal/heart/demon/pulse,
+			/obj/item/organ/internal/eyes/cybernetic/eyesofgod)
```

Comment:
This really is just stylistic, but for consistency, we have the ) on the same indentation as the declaration, after the last element.
```suggestion
			/obj/item/organ/internal/eyes/cybernetic/eyesofgod
		)
```